etc/webapi.xml

```diff
@@ -124,6 +124,20 @@
         </resources>
     </route>
 
+    <route url="/V1/adyen/orders/carts/mine/donation-campaigns" method="POST">
+        <service class="Adyen\Payment\Api\AdyenDonationCampaignsInterface" method="getCampaigns"/>
+        <resources>
+            <resource ref="self"/>
+        </resources>
+    </route>
+
+    <route url="/V1/adyen/orders/guest-carts/donation-campaigns" method="POST">
```

Comment:
Why don't we follow the Magento best practice of adding the `cartId` to the guest-user related endpoint URLs?